### ISSUE TITLE
Contact developers directly for EOL runtimes instead of asking users to do it

### ISSUE THREAD

**Issue by afranke** [flathub/flathub#2184]
`flatpak update` gives me this warning:

>  The GNOME 3.36 runtime is no longer supported as of February 13, 2021. Please ask your application developer to migrate to a supported platform.

and then gives me a list of applications. It would be way more efficient if Flathub detected such cases and sent a notification to the relevant people directly.

**Comment by Mailaender** [comment]
Please do encourage people to send pull requests to update the runtime, patch the build, and then resolve all the pedantic metadata linter problems themselves instead. This only creates entitlement. I don't get paid by anyone to do this.

**Comment by razzeee** [comment]
That command output isn't controlled by flathub. We can't remove or even change that.

**Comment by afranke** [comment]
As far as I can see, the https://github.com/flathub-infra/flatpak-builder-lint/pull/689 will tell app maintainers that an upgrade is due when they try to build against an unsupported runtime. That’s cool, but it doesn’t really address the myriad of app developers who last built their app long ago and haven’t updated them since, and who may be unaware that the version they built on has become unsupported.

**Comment by bbhtt** [comment]
Yes and there is no reason to wake multiple 100s of people up who hasn't been active at all just to tell them their runtime is EOL on every new release, which likely will not result in an update anyway (because well they aren't active).
